src/schema/index.js

```diff
@@ -53,6 +53,7 @@ type Vote {
 
 type Block {
   blockNum: Int!
+  blockTime: Int!
```

Comment:
dunt think we need this

Reply:
Ya I guess the blocks are only used for the synchronizer. Will remove it.